src/Compositor.cpp

```diff
@@ -2487,6 +2471,30 @@ PHLWINDOW CCompositor::getWindowByRegex(const std::string& regexp) {
         return nullptr;
     }
 
+    eFocusWindowMode mode = MODE_CLASS_REGEX;
+
+    std::regex       regexCheck(regexp);
+    std::string      matchCheck;
+
+    if (tRegexp.starts_with("class:")) {
```

Comment:
why all these changes and shuffling around?

Reply:
It's cause I grouped the "floating" and "tiled" checks above with "active" as those cases can return early without creating a regex object